src/App.tsx

```diff
@@ -1,25 +1,46 @@
-import React from 'react';
+// App.tsx
+import React, { useState } from 'react';
 import './App.css';
 import { getNumbers } from './utils';
+import { Pagination } from './components/Pagination';
```

Comment:
Ensure that the `Pagination` component is correctly implemented as a stateless component in `components/Pagination.tsx`. It should handle page changes and highlight the current page as specified in the task description.